repo/packages/B/bookkeeper/0/marathon.json.mustache

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "{{bookkeeper.app_id}}",
+  "cpus":{{bookkeeper.cpus}},
+  "mem": {{bookkeeper.mem}},
+  "instances": {{bookkeeper.instances}},
+  "constraints": [["hostname", "UNIQUE"]],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/bk/journal",
+        "hostPath": "/mnt/journal",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/index",
+        "hostPath": "/mnt/index",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/ledgers",
+        "hostPath": "/mnt/ledgers",
+        "mode": "RW"
+      },
+  
+      {
+        "containerPath": "persist",
+        "mode": "RW",
+        "persistent": {
+          "size": {{bookkeeper.psize}}
+        }
+      }
+    ],
+
+    "docker": {
+      "image": "{{resource.assets.container.docker.bookkeeper}}",
+      "network": "HOST"
+    },
+    
+    "env": {
+        "ZK": {{bookkeeper.zk_client}},
+        "BPORT": {{bookkeeper.service_port}}
```

Comment:
Hi @jdef, Thanks a lot for the instructions, but it seems not able to communicating the docker container via PORT0, here is my try:

in app.json file, it have ports like these:
```
    "ports": [
        3181, 9001
    ],
    "requirePorts" : true
```
It is expected a PORT0 = 3181, and a PORT1=9001, which could be referencd in Dockerfile.

In Dockerfile,
``` 
echo "port0 is : $PORT0, port1 is $PORT1"
```

This will always echo a random port0 value and a null port1 value, instead of the one in json file(3181).